bindings/python/.evergreen/test.sh

```diff
@@ -19,7 +19,8 @@ if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
              "C:/python/Python38/python.exe"
              "C:/python/Python39/python.exe"
              "C:/python/Python310/python.exe"
-             "C:/python/Python311/python.exe")
+             "C:/python/Python311/python.exe"
+             "C:/python/Python312/python.exe")
```

Comment:
Let's remove this until cffi works on 3.12 otherwise the build will be failing for everyone.

Reply:
Switched to testing on RHEL